### PR TITLE
initialize project

### LINKED ISSUE #1: sprint-1/issues-1#initialize-project
## INITIALIZE PROJECT

> Execute the command needed to start a react native project inside the repository folder.

### Tasks
- [x] initialize project

## DESIGN

> 
OLIVER 1

### PR DESCRIPTION
**Pull request by VISOSO2403**
# Description

Create React Native App to add  Sign Up design screen

## Screenshots

WhatsApp Image 2024-01-15 at 7 45 32 PM

### CODE REVIEW

File: RecervacionVuelos/Gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
```
Comment (juperameza) [inline RecervacionVuelos/Gemfile]: Why is there a Ruby file??

**Review by juperameza — APPROVED** [review]
My bad 
Lgtm 
Just remember to add reviewers in the pr

File: RecervacionVuelos/Gemfile (hunk 1 above)
Comment (VISOSO2403) [reply RecervacionVuelos/Gemfile]: It is a default file for React Native when using Mac to emulate with Ios, it contains only cocoapods version 😸

**Review by JuanequeX — CHANGES_REQUESTED** [review]
Fix the typo in your directory `recervacionVuelos` to `ReservacionVuelos`

**Review by juperameza — APPROVED** [review]

**Comment by VISOSO2403** [comment]
> Fix the typo in your directory `recervacionVuelos` to `ReservacionVuelos`

Ready, can you check again?

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**